duxy/Day6/ej1.js

```diff
@@ -0,0 +1,46 @@
+var fs = require('fs');
+var index;
+
+fs.readFile("input.txt", 'utf8', function(err, data) {
+  if (err) throw err;  
+  code(data);
+
+});
+
+function code(data){ 
```

Comment:
This method is too collapsed, all lines of code without spaces between them, is too hard to read